src/components/Projects.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div id="projects">
+    <h2 class="text-center text-3xl font-semibold text-indigo-100">
+      Our Projects
+    </h2>
+
+    <div
+      v-for="projects in ProjectData"
+      :key="projects"
+      class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 xl:grid-cols-4 gap-4"
+    >
+      <!-- CARD ITEM -->
+      <span
+        v-for="project in projects"
+        :key="project"
+        class="bg-blue-700 rounded max-w-sm overflow-hidden shadow-lg transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-105"
+      >
+        <a :href="project.link" target="_blank">
+          <img
+            :src="project.imageSrc"
+            alt="Sample Project"
+            class="w-full h-64 object-cover"
+          />
+          <h3 class="mt-3 ml-3 font-bold text-indigo-100 text-xl mb-2">
+            {{ project.name }}
+          </h3>
+          <p class="mx-3 text-indigo-100 text-base pb-3">
+            {{ project.description }}
+          </p>
+        </a>
+      </span>
+    </div>
+    <!-- END CART ITEM -->
```

Comment:
maybe we can make this a `Card.vue` component like in [Svelte's verion](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/blob/develop/src/Card.svelte). What do you think?